GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -135,6 +137,10 @@
 // Minimum time to expiration for a restored access token.
 static const NSTimeInterval kMinimumRestoredAccessTokenTimeToExpire = 600.0;
 
+// Represents a callback block that takes a `GIDUserAuth` on success or an error if the operation
+// was unsuccessful.
+typedef void (^GIDSignInCallback)(GIDUserAuth *_Nullable userAuth, NSError *_Nullable error);
```

Comment:
I'd suggest renaming this `GIDUserAuthCallback` and moving it to GIDSignIn_Private.h so that you can use it in `GIDSignInInternalOptions` and elsewhere.

Reply:
Done.